include/hip/hip_runtime_api.h

```diff
@@ -696,7 +691,7 @@ enum hipLimit_t {
 #define hipCooperativeLaunchMultiDeviceNoPreSync 0x01
 #define hipCooperativeLaunchMultiDeviceNoPostSync 0x02
 #define hipCpuDeviceId ((int)-1)
-#define hipInvalidDeviceId ((int)-2)
+#define hipErrorInvalidDeviceId ((int)-2)
```

Comment:
This change is breaking CI build.

Reply:
Yup, I did not check hipamd, so I was confused on how this is used, turns out these two are separate things:
- `hipErrorInvalidDevice` an error that is returned when the device id supplied to an api call is not valid
- `hipInvalidDeviceId` is a possible return value for `hipMemRangeAttributePreferredLocation`, ironically this is not documented anywhere.

Should be fixed now.